draft-johansson-ccwg-rfc8298bis-screamv2.md

```diff
@@ -1133,7 +1142,10 @@ end
 
 # Scale down rate slighty when the reference window is very
 # small compared to MSS
-tmp_t *= 1.0 - min(0.8, max(0.0, ref_wnd_ratio - 0.1))
+tmp_t *= 1.0 - min(0.2, max(0.0, ref_wnd_ratio - 0.1))
```

Comment:
As there are algorithmic changes, maybe add a "Changes" section that document the non-editorial changes.

Reply:
Explained in new section Algorithm changes